indico/queries/workflow.py

```diff
@@ -280,3 +281,59 @@ def __init__(
         self, workflow_id: int, files: List[str] = None, urls: List[str] = None
     ):
         super().__init__(workflow_id, files=files, urls=urls, submission=True)
+
+
+class _AddDataToWorkflow(GraphQLRequest):
+    query = """
+        mutation addDataToWorkflow($workflowId: Int!) {
+            addDataToWorkflow(workflowId: $workflowId){
+                subsetId
```

Comment:
Isn't subset id set async? What's the point of including it here?

Reply:
@nkassis so the subset_id that comes back from AddDataToWorkflow is the new subset_id that the workflow is going to be working with - do you think it's worth returning/exposing?

Reply:
Functionally, what can you do with it? If it's just extraneous then returning it wouldn't be super useful besides data management on the client side (though that might be a reason in and of itself)

Reply:
no real functionality other than information - ok without it